salt/apps/ocw/templates/webhook-publish.sh.jinja

```diff
@@ -45,24 +60,37 @@ else
     rm $retry_file
 fi
 
+cd /opt/ocw/hugo-course-publisher \
+    || error_and_exit "Can not cd to hugo-course-publisher!"
+
+orig_commit=`git rev-parse HEAD`
+log_message "Pulling hugo-course-publisher"
+git pull || error_and_exit "Can not pull hugo-course-publisher"
+new_commit=`git rev-parse HEAD`
+log_message "$orig_commit -> $new_commit"
+
+if [ $option == "full" ]; then
+    if [ "$orig_commit" != "$new_commit" ]; then
```

Comment:
Shouldn't these conditionals be swapped?

Reply:
The logic is for it to not bother checking if it's run with the "full" option. The only good stuff inside this block is where it exits the script.

Reply:
Oops! It should be `!=`, not `==` ...

Reply:
Then shouldn't it be `$option != "full"`?

Reply:
Yeah, just changed it in commit c803bb41